kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1437,6 +1437,28 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val ENGINE_FLINK_MEMORY: ConfigEntry[String] =
```

Comment:
The new configs should be added in the setting.md. you can refer docs/develop_tools/Append descriptions of new configurations to settings.md.